tests/foreman/ui/test_bookmark.py

```diff
@@ -47,12 +47,7 @@ def ui_entities(module_org, module_loc):
         # Skip the entities, which can't be tested ATM (not implemented in
         # airgun or have open BZs)
         skip = entity.get('skip_for_ui')
-        if isinstance(skip, tuple):
-            bug_type, bug_id = skip
-            if (bug_type == 'bugzilla' and bz_bug_is_open(bug_id)
-                    or bug_type == 'redmine' and rm_bug_is_open(bug_id)):
-                skip = True
-        if skip is True:
+        if skip:
```

Comment:
There are two references to bugzilla in `BOOKMARK_ENTITIES`: bz#1638781 (CLOSED ERRATA) and bz#1387569 (CLOSED WONTFIX). According to current logic, one will return False for `is_open`, while other would return True.

I think we are incorrectly skipping `ContentCredential` here.